### ISSUE TITLE
perform initial build on default branch

### ISSUE THREAD

**Issue by sellout** [garnix-io/issues#49]
When garnix is enabled for a repo, it’d be great to get an initial build without having to push further changes to the repo to trigger it.

**Comment by jkarni** [comment]
Good idea. It might be another 2-3 weeks till I get to it, since I'm finishing up a quite big feature still. Just writing this to say I'm aware and agree with the idea!
